Automattic/jetpack
# Update/Settings ui

Adds the the same wrapper of the admin to the shareing settings. 

Before:
Stats Screen
![screenshot_2018-07-25 site stats aaaaa test site will wordpress](https://user-images.githubusercontent.com/115071/43241591-e4bdd27e-9050-11e8-9a3d-ca1a8e19aa6d.png)

Shareing Settings Screen
![screenshot_2018-07-25 sharing settings aaaaa test site will wordpress](https://user-images.githubusercontent.com/115071/43241597-e596e032-9050-11e8-92df-fa59469754cb.png)

Debug screen in wp-admin
![screenshot_2018-07-25 aaaaa test site will wordpress](https://user-images.githubusercontent.com/115071/43241873-717582ec-9052-11e8-9c65-6daa97ced58f.png)

After:
![screenshot_2018-07-25 site stats enejdev wordpress](https://user-images.githubusercontent.com/115071/43241594-e52e6fac-9050-11e8-8c5e-3dfed7b07038.png)


![screenshot_2018-07-25 sharing settings enejdev wordpress](https://user-images.githubusercontent.com/115071/43241596-e560ebc6-9050-11e8-9a98-78d2b1faecda.png)

![screenshot_2018-07-25 enejdev wordpress](https://user-images.githubusercontent.com/115071/43241889-7feecf7c-9052-11e8-99ba-25de8116efd0.png)

#### Changes proposed in this Pull Request:

*

#### Testing instructions:
Check out the different wp-admin pages now. 
 Click around. Resize the window. Try in it on different devices.
- Jetpack > Stats
- Jetpack > Stats > Settings 
- Jetpack > Bulk settings > wp-admin/admin.php?page=jetpack_modules 
- Settings > Sharing 
- Jetpack (footer) Debug link
- Network Admin (need to create a MULTI SITE FOR THIS) PROTIP: (Use https://jurassic.ninja/specialops/ ) 
-- Network > Jetpack > Sites
-- Network > Jetpack > Settings



#### Proposed changelog entry for your changes:
